packages/docs/src/examples/v-combobox/misc-advanced.vue

```diff
@@ -40,7 +40,7 @@
             small
             @click="parent.selectItem(item)"
           >
-            close
+            mdi-close
```

Comment:
This will match the default chip icon:
```suggestion
            $delete
```